Makefile

```diff
@@ -105,22 +105,16 @@ else
 	@ echo "Env not defined"
 endif
 
-local-upload-data:
-	curl -vvvv -F "upload=@$(file);type=application/application/vnd.redhat.hccm.tar+tgz" \
-		-H "x-rh-identity: ${b64_identity}" \
-		-H "x-rh-request_id: testtesttest" \
-		http://localhost:${INGRESS_PORT}/api/ingress/v1/upload
-
 upload-msg-to-rosocp:
 	echo ${ros_ocp_msg} | docker-compose -f scripts/docker-compose.yml exec -T kafka kafka-console-producer --topic hccm.ros.events  --broker-list localhost:29092
 
 
 get-recommendations:
 ifdef env
 	$(eval APIPOD=$(shell oc get pods -o custom-columns=POD:.metadata.name --no-headers -n ${env} | grep ros-ocp-backend-api))
-	oc exec ${APIPOD} -c ros-ocp-backend-api -n ${env} -- /bin/bash -c 'curl -s -H "X-Rh-Identity: ${b64_identity}" -H "x-rh-request_id: testtesttest" http://localhost:8000/api/cost-management/v1/recommendations/openshift' | python -m json.tool
+	oc exec ${APIPOD} -c ros-ocp-backend-api -n ${env} -- /bin/bash -c 'curl -s -H "X-Rh-Identity: ${b64_identity}" -H "x-rh-request_id: testtesttest" http://localhost:8000/api/cost-management/v1/recommendations/openshift?start_date=2023-04-01' | python -m json.tool
```

Comment:
Is it possible to not hardcode the start_date? Its my assumption that make file should have a way to pass cmd line arguments and then those can be used?

something like:
~~~
make get-recommendations start_date=2023-08-01
~~~

Reply:
The [ros-ocp-usage.csv](https://github.com/RedHatInsights/ros-ocp-backend/blob/main/scripts/samples/ros-ocp-usage.csv) file we are using to upload data to local kafka topic for processor to upload results to recommendations populated after this date. So, I had faced an issue as a newcomer that I was not able to fetch recommendations because I didn't knew what start date was.

Thanks @upadhyeammit for pointing this out. Maybe @patilsuraj767 has something to add to this?